runtime/parachains/src/dmp.rs

```diff
@@ -254,4 +266,26 @@ impl<T: Config> Pallet<T> {
 	pub(crate) fn dmq_contents(recipient: ParaId) -> Vec<InboundDownwardMessage<T::BlockNumber>> {
 		DownwardMessageQueues::<T>::get(&recipient)
 	}
+
+	/// Raise the delivery fee factor by a multiplicative factor and stores the resulting value.
+	///
+	/// Returns the new delivery fee factor after the increment.
+	pub(crate) fn increment_fee_factor() -> FixedU128 {
+		<DeliveryFeeFactor<T>>::mutate(|f| {
+			*f = f.saturating_mul(MULTIPLICATIVE_FEE_FACTOR);
+			*f
+		})
+	}
+
+	/// Reduce the delivery fee factor by a multiplicative factor and stores the resulting value.
+	///
+	/// Does not reduce the fee factor below the initial value, which is currently set as 1.
+	///
+	/// Returns the new delivery fee factor after the decrement.
+	pub(crate) fn decrement_fee_factor() -> FixedU128 {
+		<DeliveryFeeFactor<T>>::mutate(|f| {
+			*f = InitialFactor::get().min(*f / MULTIPLICATIVE_FEE_FACTOR);
```

Comment:
Don't we want max, not min here otherwise it will instantly reset to the InitialFactor?
```suggestion
			*f = InitialFactor::get().max(*f / MULTIPLICATIVE_FEE_FACTOR);
```